### ISSUE TITLE
Rulers don't work in splitview

### ISSUE THREAD

**Issue by JeffryBooher** [lkcampbell/brackets-ruler#37]
Hey Lance,
I tested your Column Ruler extension with splitview (Jeff/Splitview-factory) and it doesn't work as designed.  

1) primary ruler extends over secondary view
2) no ruler for secondary view

You can test with the jeff/splitview-factory branch

**Comment by demisx** [comment]
+1 The ruler disappears once I go into 2 column view.

**Comment by Hirse** [comment]
Not knowing that something like this already existed, I have written the [brackets-print-margin extension](https://github.com/Hirse/brackets-print-margin) to add a column guide. I called the guide line "print margin" as in the Eclipse preferences though.
I don't have the ruler on top, but the line is also displayed in multiple editors in splitview by adding a background to `.CodeMirror-lines`.

Maybe you could do something similar to at least display the guide in splitview.

**Comment by GAZ082** [comment]
@Hirse Thank you!

**Comment by lkcampbell** [comment]
@Hirse, agreed. This extension has been a nightmare to maintain. I'm going to toss the entire ruler interface in the trash and just rewrite this extension with user preferences, basically a thin layer over the CodeMirror addon.
